tests/test_time.py

```diff
@@ -599,27 +613,27 @@ def test_precisedelta_custom_format(val, min_unit, fmt, expected):
         ),
     ],
 )
-def test_precisedelta_suppress_units(val, min_unit, suppress, expected):
+def test_precisedelta_suppress_units(
+    val: dt.timedelta, min_unit: str, suppress: list[str], expected: str
+) -> None:
     assert (
         humanize.precisedelta(val, minimum_unit=min_unit, suppress=suppress) == expected
     )
 
 
-def test_precisedelta_bogus_call():
-    assert humanize.precisedelta(None) is None
-
+def test_precisedelta_bogus_call() -> None:
     with pytest.raises(ValueError):
         humanize.precisedelta(1, minimum_unit="years", suppress=["years"])
 
     with pytest.raises(ValueError):
         humanize.naturaldelta(1, minimum_unit="years")
 
 
-def test_time_unit():
+def test_time_unit() -> None:
     years, minutes = time.Unit["YEARS"], time.Unit["MINUTES"]
     assert minutes < years
     assert years > minutes
     assert minutes == minutes
 
     with pytest.raises(TypeError):
-        years < "foo"
+        assert years < "foo"
```

Comment:
Assume this change is to avoid some sort of linter complaining about an expression that does nothing.

But the assertion implies you want to check the truthish of the expression.

What about 
```suggestion
        _ = years < "foo"
```

Using `_` as the generic "We don't care about this" variable. Is the linter happy with that?

Reply:
Yeah, just a silent warning in PyCharm:

![image](https://user-images.githubusercontent.com/1324225/166674267-4e90594c-7afa-466a-87b2-80cfb3d5bc65.png)


![image](https://user-images.githubusercontent.com/1324225/166674251-a2567bab-cc73-4503-b74a-2d92e08005a0.png)

It's happy with your suggestion:

![image](https://user-images.githubusercontent.com/1324225/166674338-b88ccfed-85f7-4dcc-b1e4-7e3e7e46850d.png)